sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-helper.ts

```diff
@@ -31,24 +39,31 @@ export async function createInstitutionLocations(
     "DECL",
     e2eDataSources,
   );
-
   const institutionLocationSKIP = await findOrCreateInstitutionLocation(
     institution,
     "SKIP",
     e2eDataSources,
   );
-
   const institutionLocationFAIL = await findOrCreateInstitutionLocation(
     institution,
     "FAIL",
     e2eDataSources,
   );
+  // Institution location to test disbursement with multiple detail records.
+  const institutionLocationMULT = await findOrCreateInstitutionLocation(
+    institution,
+    "MULT",
+    e2eDataSources,
+    // Multiple integration contacts.
+    { integrationContacts: [faker.internet.email(), faker.internet.email()] },
```

Comment:
how about moving this logic to `findOrCreateInstitutionLocation`  and the contact type will depend on the flag?

Reply:
I feel that letting the consumer replace the default data that the method `findOrCreateInstitutionLocation ` creates will have more flexibility at consumer side to use it. e.g. what if consumer wants the data to be just [] or an invalid data in future.

Reply:
it can be achieved with the same flag also right

Reply:
Sorry, I am not able to follow this. My assumption on hearing a flag was a boolean value passed to the method and and method creating data1 or data2 based on true or false of the boolean. But what If we may need more that 2 types of data ?

Am I missing something here.

Reply:
My thought was a flag (  probably a string or enum like 'MULT', 'SINGLE', or even a boolean) which decides the number of contact on `findOrCreateInstitutionLocation`, and if the option is not present, then contacts can be null

Reply:
Had a discussion with same on @andrewsignori-aot and as per our dev discussion in one of the previous PR, using `options?: { initialValues?: Partial<InstitutionLocation> }` to set the initial values(integrationContacts here).

Reply:
To be clear, in the last PR, we talked about having `initialValues?: Partial<InstitutionLocation>` and we have an example of it in the code already but not inside options.
Yesterday during a chat we @dheepak-aot proposed to it be inside options, which I think is a good idea to follow from now on.

Reply:
okay, so moving forward the plan is to add the `initialValues` in the options param